WcaOnRails/config/locales/en.yml

```diff
@@ -648,6 +648,12 @@ en:
         '5': "The avatar must not include texts other than regular background texts."
         '6': "The avatar must be submitted in correct orientation."
         '7': "The avatar must not contain offensive content (e.g. offensive gestures)."
+      delegate_avatar_guidelines:
+        title: "If you are a delegate, please observe also the following guidelines:"
```

Comment:
Just a small grammar mistake: "If you are a Delegate, please also observe the following guidelines"

Reply:
Actually, given we show this if and only if the user is `any_kind_of_delegate?`, we could get rid of the conditional sentence, right? Maybe something along the lines of "Additionally, the following guidelines apply to delegates:"?

Reply:
It seems from the code that *this part* would be shown to everyone. Is it shown to Delegates only?

Reply:
Hmm, which part of the code are you referring to? Isn't it this one:

```erb
<% if current_user.any_kind_of_delegate? %>
  <br /><strong><%= t('.delegate_avatar_guidelines.title') %></strong>
  <ul>
    <% t('.delegate_avatar_guidelines.paragraphs').values.each do |guideline| %>
      <li><%= guideline %></li>
    <% end %>
  </ul>
<% end %>
```

Reply:
Hmm, it might I'm a bit unsure. This is only shown to Delegates right?

Reply:
Yeah, I didn't look properly through it. That should show this only to Delegates.
I also think another title would be better, since it is only shown to the Delegates.

Reply:
I think it would be best to change it to "Additional guidelines for Delegates".

Reply:
Actually, now I wonder whether it shouldn't be `@user` instead of `current_user`. At the moment it is just fine if we think of someone going to his profile page, but it may be the case that someone else (e.g. a WRT member) goes to someone else's profile. We want to show the guidelines if they apply for the user being viewed, not the one viewing.

Reply:
It makes sense indeed. I updated the code.